package.json

```diff
@@ -14,40 +13,18 @@
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
-    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.18.9",
     "@babel/preset-env": "^7.18.10",
     "@babel/register": "^7.0.0",
-    "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-commonjs": "^22.0.2",
-    "@rollup/plugin-inject": "^4.0.4",
-    "@rollup/plugin-node-resolve": "^13.3.0",
     "@semantic-ui-react/css-patch": "^1.1.2",
     "@symfony/webpack-encore": "^3.1.0",
-    "babel-plugin-fast-async": "^6.1.2",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "dedent": "^0.7.0",
     "eslint": "^8.23.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-import-resolver-babel-module": "^5.3.1",
     "eslint-plugin-import": "^2.26.0",
-    "fast-async": "^6.3.8",
-    "gulp": "^4.0.2",
-    "gulp-chug": "^0.5.1",
-    "gulp-concat": "^2.6.1",
-    "gulp-debug": "^4.0.0",
-    "gulp-if": "^3.0.0",
-    "gulp-livereload": "^4.0.2",
-    "gulp-order": "^1.2.0",
-    "gulp-sass": "^5.1.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulp-uglifycss": "^1.1.0",
-    "merge-stream": "^2.0.0",
-    "rollup": "^2.79.0",
-    "rollup-plugin-terser": "^7.0.2",
     "sass": "^1.54.8",
     "sass-loader": "^13.0.0",
-    "upath": "^2.0.1",
```

Comment:
Just a thought out loud,
It's interesting, that gulp used [`upath`](https://www.npmjs.com/package/upath#why-), while webpack is fine with npm's `path` package.